src/tool/hpcrun/gpu/nvidia/cupti-activity-translate.c

```diff
@@ -456,6 +456,7 @@ convert_branch
 }
 
 
+__attribute__((unused))
```

Comment:
This function is only "used" in a comment in `cupti_activity_translate`. If it's important to keep this code around for later, I would recommend an `#if 0` and a comment explaining why we're keeping dead code around like this.

And then revise `cupti_activity_translate` accordingly to minimize/remove the comment there and handle `CUPTI_ACTIVITY_KIND_SYNCHRONIZATION` as a proper case.